docs/knowledge_base/policy_phylum.md

```diff
@@ -0,0 +1,21 @@
+# Phylum Policies
+
+Out of the box, projects and groups have access to about two dozen of policy regos authored by Phylum. 
```

Comment:
I think we should drop "regos" here. Most people aren't familiar with that particular phrase.

```suggestion
Out of the box, Phylum provides roughly two dozen pre-defined policies that allow users to specify what their organization's acceptable use policies are for open source packages
```

Reply:
yeah... i agree.